docs/docs/basics/background-mode.md

```diff
@@ -4,44 +4,63 @@ sidebar_position: 4
 
 # Background Mode
 
-React Native Track Player supports playing audio while your app is in the background on all supported platforms.
+React Native Track Player supports playing audio while your app is in the
+background on all supported platforms.
 
 ## Android
-Background audio playback works right out of the box. By default, the audio will continue to play, not only when the app is suspended in the background, but also after the app is closed by the user. If that is not the desired behavior, you can disable it with the `android.appKilledPlaybackBehavior` property in `updateOptions`. 
+Background audio playback works right out of the box. By default, the audio will
+ontinue to play, not only when the app is suspended in the background, but also
+after the app is closed by the user. If that is not the desired behavior, you
+can disable it with the `android.appKilledPlaybackBehavior` property in
+`updateOptions`.
```

Comment:
Do we need these linebreaks?

Reply:
@dcvz what do you think?

Reply:
FWIW, I do think some character limit is a good thing (maybe 80 is too low). Otherwise you'll get code that will have lines that run on to infinity (logical evaluation expressions in conditionals/if statements is a good example).